pii_data.py

```diff
@@ -45,11 +45,14 @@ def has_ipv6(self):
         return False
         return None
 
-    def has_name(self):
-        match = re.search(r'[A-Z][a-z]{1,}\s[A-Z][a-z]{1,}', self)
-        if match:
-            return True
-        return False
+    def has_name(self, anonymize=False):
+        newstr, count1 = re.subn(r'(?<!\d\s)(?<=\s)[A-Z][a-z]{1,}\s[A-Z][a-z]{1,}', '[name]', self)
```

Comment:
The only thing preventing this from working is the positive look-behind (?<=\s) is looking for a space that needs to be present.  That means:
```
 Joe Jones
```
matches but 
```
Joe Jones
``` 
does not because there isn't a space in front of it. If you remove that look-behind it would work.

Reply:
```
(?<!\d\s)[A-Z][a-z]{1,}\s[A-Z][a-z]{1,}
```
You could write this as 
```
(?<!\d )[A-Z][a-z]+ [A-Z][a-z]+
```
Note that I changed the {1,} to + since they are equivalent and I replaced \s with an actual space - this prevents matching a new line, tab, or other characters you might not want to match

Reply:
Thank you for the information. Also done.